src/datapunt_processing/extract/download_tables_from_dokuwiki_to_json.py

```diff
@@ -0,0 +1,170 @@
+import argparse
+import requests
+import json
+import os
+
+from bs4 import BeautifulSoup
+
+
+def getPage(url):
+    data = requests.get(url).text
+    return data
+
+
+def getRows(headers, row):
+    item = []
+    cells = row.find_all(['th', 'td'])
+    for cell in cells:
+        value = cell.get_text().strip()
+        if cell.a:
+            url_name = cell.a['href']
+            if url_name[:5] == '/doku':
+                url_name = 'https://dokuwiki.datapunt.amsterdam.nl' + url_name
+            value = {"naam": value, "url": url_name}
```

Comment:
I would recommend moving "https://dokuwiki.datapunt.amsterdam.nl"  to an environment variable or startup parameter